src/mixins/add-edit-org-person-mixin.ts

```diff
@@ -259,6 +284,42 @@ export default class AddEditOrgPersonMixin extends Vue {
     this.$refs.addPersonOrgForm && this.$refs.addPersonOrgForm.validate()
   }
 
+  protected swapIsLookupBusiness (): void {
+    this.orgPerson.isLookupBusiness = !this.orgPerson?.isLookupBusiness
+  }
+
+  /**
+   * BusinessLookup component event handler - called when it is rendered and when
+   * the user undo selected business.
+   */
+  protected resetBusinessDetails (): void {
+    this.updateBusinessDetails(EmptyBusinessLookup)
+  }
+
+  /**
+   * BusinessLookup component event handler - called when it is rendered and when
+   * the user has selected a business.
+   */
+  protected async updateBusinessDetails (businessLookup: BusinessLookupIF): Promise<void> {
```

Comment:
It is possible for `businessLookup` to be null or undefined?

Reply:
No. Undo will set EmptyBusinessLookup